gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageFileSystem.java

```diff
@@ -76,161 +36,23 @@
  *     href="https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-common/filesystem/index.html">Hadoop
  *     FileSystem specification.</a>
  */
-public class GoogleCloudStorageFileSystem {
+public interface GoogleCloudStorageFileSystem {
 
-  private static final GoogleLogger logger = GoogleLogger.forEnclosingClass();
+  GoogleLogger logger = GoogleLogger.forEnclosingClass();
```

Comment:
This makes logger a public field - it will be better to just remove it and log statements in default methods.

Reply:
Removed the logging, as it was already logged in the called method